tests/dialects/test_riscv.py

```diff
@@ -0,0 +1,13 @@
+from xdsl.utils.test_value import TestSSAValue
+from xdsl.dialects import riscv
+
+
+def test_add_op():
+    a1 = TestSSAValue(riscv.RegisterType(riscv.Register()))
```

Comment:
I think you can directly import `RegisterType` in that file, as it's going to make things more concise here.
But not a strong feeling.

Reply:
There will be some sugar added in subsequent pull requests, I'd rather stick to the verbose version for now